MSAL/src/requests/MSALSilentRequest.m

```diff
@@ -182,13 +181,13 @@ - (MSIDTokenRequest *)tokenRequest
 
 - (BOOL)isServerUnavailable:(NSError *)error
 {
-    if (![error.domain isEqualToString:MSIDHttpErrorCodeDomain])
+    if (![error.domain isEqualToString:MSALErrorDomain])
     {
         return NO;
     }
     
-    NSInteger responseCode = [[error.userInfo objectForKey:MSIDHTTPResponseCodeKey] intValue];
-    return error.code == MSIDErrorServerUnhandledResponse && responseCode >= 500 && responseCode <= 599;
+    NSInteger responseCode = [[error.userInfo objectForKey:MSALHTTPResponseCodeKey] intValue];
+    return error.code == MSALErrorUnhandledResponse && responseCode >= 500 && responseCode <= 599;
```

Comment:
:+1: I have the same change in my PR, handling MSID error here doesn't make sense at all :)